online-workflow/src/main/java/eu/itesla_project/online/StateAnalyzer.java

```diff
@@ -430,7 +430,7 @@ private boolean computePostContingencyState(Network network, String stateId, Con
             network.getStateManager().setWorkingState(postContingencyStateId);
             // apply contingency to post contingency state
             logger.info("{}: applying contingency {} to post contingency state {}", stateId, contingency.getId(), postContingencyStateId);
-            contingency.toTask().modify(network);
+            contingency.toTask().modify(network, null);
```

Comment:
The computationManager have been passed by the caller.

Reply:
Have been passed? I cound not find in StateAnalyzer.
I added it in the last commit.